host/frontend/host-orchestrator/orchestrator/instancemanager.go

```diff
@@ -133,22 +137,47 @@ func NewLaunchCVDProcedureBuilder(
 		},
 		stageCreateArtifactsDir: &StageCreateDirIfNotExist{Dir: paths.ArtifactsRootDir},
 		stageCreateHomesDir:     &StageCreateDirIfNotExist{Dir: paths.HomesRootDir},
+		// Fetch CVD stages will be reused to avoid downloading the artifacts of the same
+		// target multiple times.
```

Comment:
This comment should be in the type declaration.

Reply:
Done